Assets/_Scripts/FightingEntity.cs

```diff
@@ -36,8 +43,18 @@ public class FightingEntity : MonoBehaviour
 		SetStats(data.stats);
 		SetJob(data.job);
 		_ai = new FightingEntityAI(this);
+        this.targetName = GameManager.Instance.turnController.field.GetTargetNameFromIndex(index);
 	}
 
+    public void InitializePosition(FighterSlot slot) {
```

Comment:
As far as the entity is concerned, it shouldn't really need to know where it is. We should do this transform update in the FighterSlot and not in the FightingEntity.

Reply:
We need to know the slot to manage the message box anyway, so this does it all in one

Reply:
On second thought, this function can probably be put in FighterSlot

Reply:
Made the change